### PR TITLE
Make it language aware when needed.

### PR DESCRIPTION
**Pull request by freshface**
## Type

- Enhancement

## Resolves the following issues

When you get a tag or tags you can make it language aware.

### CODE REVIEW

**Comment by carakas** [comment]
@freshface can you look at the feedback of @jeroendesloovere

**Comment by freshface** [comment]
I changed it.

File: src/Frontend/Modules/Tags/Engine/Model.php [hunk 1]
```diff
@@ -12,6 +12,7 @@
 use Frontend\Core\Engine\Exception as FrontendException;
 use Frontend\Core\Engine\Model as FrontendModel;
 use Frontend\Core\Engine\Navigation as FrontendNavigation;
+use Backend\Core\Language\Locale;
```
Comment (carakas) [inline src/Frontend/Modules/Tags/Engine/Model.php]: you need to use the frontend version :)

File: src/Frontend/Modules/Tags/Engine/Model.php [hunk 2]
```diff
@@ -92,11 +93,14 @@ public static function getAll()
...
      * @return array
      */
-    public static function getForItem($module, $otherId)
+    public static function getForItem($module, $otherId, $language = null)
     {
         $module = (string) $module;
         $otherId = (int) $otherId;
 
+        // redefine language
+        $language = ($language !== null) ? (string) $language : Locale::workingLocale();
```
Comment (carakas) [inline src/Frontend/Modules/Tags/Engine/Model.php]: this method will also need to change to the frontend equivalent

**Comment by carakas** [comment]
@freshface some new feedback

**Comment by freshface** [comment]
I fixed it.

**Pull request merged**